infrastructure/testdriver/send_keys.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>TestDriver click method</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+
+<input type="text" id="text">Button</button>
+
+<script>
+async_test(t => {
+  let text_box = document.getElementById("text");
+  test_driver
+    .send_keys(text_box, "Hello, wpt!")
+    .then(() => t.done())
```

Comment:
Can we check this actually worked here?

Reply:
This still seems like a reasonable comment, @kereliuk!

Reply:
No, GitHub is just showing me random diffs that aren't of the right commit when I first click here. o_O